procrustes/utils.py

```diff
@@ -336,3 +339,55 @@ def _check_arraytypes(*args):
         raise TypeError("Matrix inputs must be NumPy arrays")
     if any(x.ndim != 2 for x in args):
         raise TypeError("Matrix inputs must be 2-dimensional arrays")
+
+
+class OptResult(dict):
```

Comment:
It is more informative to call it `ProcrustesResult` probably.

Reply:
I have renamed it as `ProcrustesResult`.